packages/suite/src/reducers/suite/desktopUpdateReducer.ts

```diff
@@ -10,6 +10,17 @@ import { UpdateInfo, UpdateProgress, UpdateWindow } from '@suite-types/desktop';
  * - not-available: Currently on the latest version
  * - downloading: Currently downloading the latest version
  * - ready: Latest version is downloaded and going to be installed on the next restart
+ */
+export enum UpdateState {
+    Checking = 'checking',
```

Comment:
nit: We have ~7 enums defined so far, and most of them use `UPPER_SNAKE_CASE` for keys. I personally don't like it, but let's keep it consistent.  (I would prefer camelCase for the keys and basically for all constants and PascalCase just for names of enums, interfaces, React components and so on...)

Reply:
I did it according to TS docs https://www.typescriptlang.org/docs/handbook/enums.html and the way most people in `Medium` articles do it. I agree with you that also a lot of people uses PascalCase for names and UPPER_CASE for keys. However, you do not like it, so I would you the TS docs style.

We have a mix of everything in the codebase (even a mix in one enum). 
```
export enum Flags {
    None = 0,
    hasEmail = 1 << 0,
    hasBookmark = 1 << 1
}
```

So, I would leave it like this. We have only a few enums in the codebase and I think we should use them more in the future. Before we start using it, we can have a discussion and unite all the existing enums to the style we decide.

Reply:
Ok, I just created a low-hanging-fruit issue for it: https://github.com/trezor/trezor-suite/issues/4504